doc/ir-semantics.text

```diff
@@ -108,18 +118,30 @@ Update (partial) function, returns a S:
        = step CC[v] (P, T, H, E*, C*)
     when v := eval P H E e
 
+  step   CC[call f (v, ...)]
+         (P, T, H, E*, C*)
+       = (P, T, H, E* E', C* C)
```

Comment:
see above, the whole C and CC business is causing confusion. `CC` is really only used to pattern match in `step`.